src/components/GameController.tsx

```diff
@@ -41,40 +42,77 @@ export default class GameController extends PureComponent<Props> {
     }
 
     switch (applicationState && applicationState.constructor) {
-      case playerAStates.ReadyToSendPreFundSetupA:
+      case playerA.ReadyToSendPreFundSetupA:
```

Comment:
It'd be nice if we could pull out some of this switch logic so the component isn't so verbose. Maybe a simple mapper that maps a `state` -> `message`. Probably something for a later date when the UI evolves.